internal/appconfig/config.go

```diff
@@ -147,3 +147,11 @@ func (c *Config) DockerBuildTarget() string {
 	}
 	return c.Build.DockerBuildTarget
 }
+
+func (c *Config) MountsDestination() string {
+	if c.Mounts == nil {
+		return ""
+	} else {
+		return c.Mounts.Destination
+	}
```

Comment:
wdyt if we follow the same pattern than the other getters right above this one?

```suggestion
	if c == nil || c.Mounts == nil {
		return ""
	} 
	return c.Mounts.Destination
```